server/controllers/authController.api.js

```diff
@@ -25,11 +25,12 @@ exports.login = (req, res) => {
   const { email, password } = req.body;
   User.findOne({ email })
     .populate('projects')
+    .populate('investments')
     .exec((err, user) => {
       if (err) {
         mongoDbErrorHandler(err, res, 400);
       } else if (user && user.comparePassword(password)) {
-        const { name, email, _id, projects, about, location, profilePic } = user;
+        const { name, email, _id, projects, about, location, profilePic, investments } = user;
         const token = encodeToken({ name, email, _id, about, location, profilePic });
         return res.status(200).json({
```

Comment:
It's because I initially had it for setting the user info like the profile pic when I was doing the edit profile. I made a ticket for fixing this issue that could be done later